README.md

```diff
@@ -10,15 +10,21 @@ Before you do anything else, you need to verify what modules should be bundled w
  * webservices.rest
  * owa
  * fhir
- 
+
 If any of these modules have significant recent commits that have not been released yet, then consider releasing them before you do the platform release.
 
+### What OWAs to include?
+
+The platform also comes bundled with some Open Web Apps (OWAs). Verify that the following OWAs are bundled with this version of OpenMRS Platform. At the time of writing these include:
```

Comment:
Can you correct the grammar? "version of OpenMRS Platform" does not look correct. 😊

Reply:
Also do some punctuations where necessary. For instance, At the time of writing, these include